test/cri-containerd/container_test.go

```diff
@@ -250,6 +251,7 @@ func Test_RunContainer_VirtualDevice_GPU_LCOW(t *testing.T) {
 				// if a given gpu device needs more, this test will fail to create the container
 				// and may hang.
 				"io.microsoft.virtualmachine.computetopology.memory.highmmiogapinmb": "64000",
+				"io.microsoft.virtualmachine.lcow.bootfilesrootpath":                 "C:\\ContainerPlat\\LinuxBootFiles\\nvidiagpu",
```

Comment:
Do we hard code paths in tests like this anywhere else?

Reply:
I don't see any occurrences than what is in this PR. I see the need to reference this path (as well as shimdiag below), but this will impose a restriction on this test suite that the package be installed to `C:\ContainerPlat`. Let's at least define a single constant in main.go for this path, so all the references can be changed easily.

Reply:
Sounds good. I wasn't sure the best way to handle that.